### PR TITLE
Throw ValidationError if value exceeds min_value or max_value

### PR DESCRIPTION
**Pull request by fidelitas**
Pull request #15 reveals missing field validation. Added validation by overriding validate() in MeasurementField.

### CODE REVIEW

**Comment by coddingtonbear** [comment]
@codingjoe -- I can look into this later if you aren't already planning to; just let me know!

**Comment by codingjoe** [comment]
@coddingtonbear: @fidelitas is my college, he found this bug. I'm getting payed to review his code, therefore I'm happy to review his code. Feel free to leave some comments tho.

File: django_measurement/models.py [hunk 1]
```diff
@@ -2,9 +2,8 @@
 from __future__ import absolute_import, unicode_literals
 
 import six
-from django.core.exceptions import ValidationError
+from django.core import exceptions
```
Comment (codingjoe) [inline django_measurement/models.py]: Why change it to module import. Since you're still only using `ValidationError`

File: tests/models.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 from django.db import models
-from measurement import measures
-
 from django_measurement.models import MeasurementField
+from measurement import measures
```
Comment (codingjoe) [inline tests/models.py]: What is not right.

Comment (codingjoe) [inline tests/models.py]: Wrong sorting.

File: tests/test_fields.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import pytest
+from django.core.exceptions import ValidationError
 from django.utils import module_loading
 from django.utils.encoding import force_bytes, force_text
 from measurement import measures
-
```
Comment (codingjoe) [inline tests/test_fields.py]: should remain

**Comment by codingjoe** [comment]
@fidelitas I left you a couple of comments

File: django_measurement/models.py [hunk 4]
```diff
@@ -118,6 +119,54 @@ def formfield(self, **kwargs):
         defaults.update(self.widget_args)
         return super(MeasurementField, self).formfield(**defaults)
 
+    def validate(self, value, model_instance):
```
Comment (codingjoe) [inline django_measurement/models.py]: why did you copy this code?

**Comment by codingjoe** [comment]
@fidelitas let's talk about this tomorrow please.

**Comment by codingjoe** [comment]
After discussion, we decided, that it makes no sense to add a undocumented feature for `IntegerField` to a `FloatField` subclass.

**Pull request closed without merging**